### DISCUSSION TITLE
Welcome to principes-mec Discussions!

### DISCUSSION THREAD

**Discussion by houssamkhalifeh** [VilledeMontreal/MontrealEnCommun#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by houssamkhalifeh** [comment]
Salut Jérémie  @psionikangel,

J'ai lancé les discussions pour faciliter/accélérer les communications entre les contributeurs.
Si je me rappelle bien, tu as créé un team Montréal en commun. Je n'arrive pas à le retrouver.
Mon but est d'envoyer des invitations directement à cette équipe et d'ajouter les repos aussi de telle sorte que les membres de l'équipe (utilisateurs externes) soient capables de voire les repos de l'équipe en un seul endroit.
J'étais capable de faire ça avec une organisation que j'ai créé mais je n'arrive pas à le faire ici. 

Aurais-tu une idée?

Merci

**Reply by psionikangel** [reply]
Bonjour Houssam,

La fonctionnalité des "teams" dans Github ne semble pas pouvoir contenir des collaborateurs externes, seulement des membres de l'organisation. Il faut donc ajouter les collaborateurs externes à la pièce sur les dépots et leur assigner les permissions nécessaires.

Si tu as une liste de comptes Github on peut s'organiser une petite réunion et je pourrai les ajouter pour toi. Remarque que cette limitation (pas de collaborateurs externes dans les teams) vient peut-être changer tes plans au niveau de ce que tu voulais mettre en place pour la communication. On peut voir quelles options s'offrent à nous dans cette même réunion.

**Reply by houssamkhalifeh** [reply]
Merci Jérémie, je vais leurs demander de m'envoyer leurs comptes GitHub. On pourrait se booker un appel conférence pour voir c'est la meilleure façon de procéder. Idéalement, on aimerait avoir un maximum de flexibilité sans trop perturber le fonctionnement de l'espace de la ville.
à suive..

**Reply by psionikangel** [reply]
👍
